src/app/components/drawer/DrawerRail.js

```diff
@@ -48,14 +49,29 @@ const DrawerRail = (props) => {
 
   const {
     drawerOpen,
+    drawerType,
+    onDrawerTypeChange,
     onDrawerOpenChange,
     team,
     currentUserIsMember,
   } = props;
 
-  const setDrawerOpenChange = () => {
-    onDrawerOpenChange(!drawerOpen);
-    window.storage.set('drawer.isOpen', !drawerOpen);
+
+  const setDrawerOpenChange = (newDrawerType) => {
+    const currentDrawerType = drawerType;
+    // if drawer is open and we click on the other button, change it
+    if (drawerOpen && currentDrawerType !== newDrawerType) {
+      onDrawerTypeChange(newDrawerType);
+      onDrawerOpenChange(drawerOpen);
```

Comment:
If it didn't change, is this call really needed? I'm thinking about useless renders.

Reply:
:white_check_mark: